README.md

```diff
@@ -163,6 +163,25 @@ Source objects with the same API/login credentials, but different regions.
 - region_name
   - This could be `New York 3`, `San Francisco 1`, etc.
 
+#### Joyent
+- proxy_port
+  - What port is exposed for proxy services on the droplet image.
+- username
+  - The username used to log into the Joyent Account
+- password
+  - The password associated with the Joyent username
+  datacenter
```

Comment:
Tiny formatting issue: looks like this line and the next one are missing their bullets (`-`s).